src/mpid/ch4/errnames.txt

```diff
@@ -6,6 +6,8 @@
 **ch4|badmsgtype:request contained an invalid message type
 **ch4|badreqtype %d:request contained an invalid request type (%d)
 **ch4|badreqtype:request contained an invalid request type
+**ch4|badstartreq %d:start an invalid request kind (%d)
+**ch4|badstartreq:start an invalid request kind
```

Comment:
Do we really need a different message? They read the same to me.

Reply:
Do you mean `**ch4|badreqtype` v.s. `**ch4|badstartreq`? The former checks persistent req internal type `MPIDI_PREQUEST(preq, p_type)`, but the latter checks `req->kind`.

Reply:
I see.